CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Security
 - Upgrade github-pages in docs/Gemfile to resolve CVE-2021-28834 in kramdown dependency [cyberark/conjur#2099](https://github.com/cyberark/conjur/issues/2099)
 
+### Added
+- Rake task for password strength validation. Can be called with `rake account:validate_password_strength[examplePasswordToCheck]`
```

Comment:
if i run this from the command line do i need to add quotes surrounding the password?
i.e should it be `rake account:validate_password_strength["examplePasswordToCheck"]`?

can you please verify this?

Reply:
its password now you are right

Reply:
i actually referred to the need of the quotes around `examplePasswordToCheck` but yes, we also need to change it to password :)

Reply:
Done